src/components/jogging/JoggingStore.ts

```diff
@@ -141,6 +130,18 @@ export class JoggingStore {
         ),
       ])
 
+    // Setting mode control makes jogging startup slightly faster
+    // on physical robots
+    // https://wandelbots.slack.com/archives/C06VA4J59PF/p1725523765976109?thread_ts=1725464963.859559&cid=C06VA4J59PF
```

Comment:
nit pick: References to internal documents might not be beneficial to external users

Reply:
It's probably fine; we expect the vast majority of external usage will be as a package and not reading the code itself

(unless there's a security issue of course, that's a different story 😄 )